docs/security-tls-authentication.md

```diff
@@ -136,7 +136,7 @@ var client = PulsarClient.Builder()
 
 ## Configure TLS authentication in CLI tools
 
-[Command-line tools](reference-cli-tools.md) like [`pulsar-admin`](/tools/pulsar-admin/), [`pulsar-perf`](reference-cli-tools.md), and [`pulsar-client`](reference-cli-tools.md) use the `conf/client.conf` config file in a Pulsar installation.
+[Command-line tools](reference-cli-tools.md) like [`pulsar-admin`](https://pulsar.apache.org/reference/#/next/pulsar-admin/), [`pulsar-perf`](https://pulsar.apache.org/reference/#/next/pulsar-perf), and [`pulsar-client`](https://pulsar.apache.org/reference/#/next/pulsar-client/) use the `conf/client.conf` config file in a Pulsar installation.
```

Comment:
Commented in https://github.com/apache/pulsar/issues/19491#issuecomment-1428991305

Reply:
Agreed. I will also double-check and fix all the broken links that have been warned during the local build for 2.8.x and later versions.

Reply:
Thank you!